### ISSUE TITLE
[Suggestion] Method of notifying or updating expired cookie

### ISSUE THREAD

**Issue by FlamingFox911** [torikushiii/hoyolab-auto#42]
Hoyolab cookies do not last forever, so ideally there should either be a notification that let's the person know if the cookie has expired, or even better, a method to allow the user directly or indirectly update their cookie. Note that for the latter, it may require user interaction due to captcha.

And I am totally not making this suggestion because I got three notifications of code redemption failures caused by not being logged in, even though I had the cookie setup in the config... 👀

**Comment by FlamingFox911** [comment]
I had a version that was from 8 days ago. I pulled the latest changes at around 8:36 AM. Then I got another notification afterwards saying "Code Redeem Failed" again, but with fixed text.
![image](https://github.com/user-attachments/assets/ddc6a3a8-22d6-4406-9834-5748d337db85)
Probably what happened is my cookie expired before I updated my code, and it never had the opportunity to update again. I'll update again if I get another successful code redemption. If not, I'll copy my logs before I rebuild.
